src/resources/Incorporations/benefitCompany.ts

```diff
@@ -10,7 +10,6 @@ export const BenefitCompanyResource: IncorporationResourceIF = {
   description: 'This company is a benefit company and, as such, has purposes that include conducting its ' +
     'business in a responsible and sustainable manner and promoting one or more public benefits.',
   statement: null,
-  nameRequestType: NameRequestTypes.BC,
```

Comment:
I'm not sure what the original plan for this field was, but I couldn't find anywhere it's used.